src/bench/bench.cpp

```diff
@@ -8,98 +8,135 @@
 #include <assert.h>
 #include <iostream>
 #include <iomanip>
+#include <algorithm>
+#include <regex>
```

Comment:
Don't forget to `#include <numeric>` for `std::accumulate`.

Reply:
thanks, updated!